docs/links.md

```diff
@@ -9,7 +9,9 @@
 - [Synthesia (9.0)](https://www.synthesiagame.com/downloadSynthesia?product=Synthesia&platform=win_sa&version=9) - Latest free version of Synthesia.
 - [Ultralight MIDI Player](https://pipiraworld.web.fc2.com/ump/en.html) - Alternative to PFA, made with Java. Can be used for rendering MIDI files.
 - [Zenith](https://github.com/arduano/Zenith-MIDI/) - ⭐ MIDI renderer, made with C#, alternative to UMP.
-- [Kiva](https://github.com/arduano/Kiva/) - ⭐ MIDI player, alternative to PFA.
+- [Kiva](https://github.com/arduano/Kiva/) - MIDI player, alternative to PFA. (Replaced by Wasabi)
+- [Wasabi](https://github.com/arduano/wasabi) - ⭐ MIDI player, alternative to PFA, while focusing on performance
+- [Chikara](https://github.com/Kaydax/Chikara) - MIDI player (Replaced by Wasabi)
```

Comment:
Huh, since when this is replaced? I must missed something big.

If Wasabi really replaced both of them, best to put it before the two.

Reply:
Yes, this has been discussed internally, and if asked we always prompt people to use Wasabi over Kiva or Chikara as both are not being worked on and been surpassed by Wasabi